### ISSUE TITLE
Output is truncated

### ISSUE THREAD

**Issue by george-polevoy** [Renset/macai#15]
**Describe the bug**
Output is truncated

**To Reproduce**
I don't know how to reproduce, but I can show the entire message returned, as I could copy it using context menu (feature of 0.1.12)

Truncated output:

Certainly! Here is an example AST for a possible implementation of the `distinctive features` function as part of the previous AST:

Full output:

````
Certainly! Here is an example AST for a possible implementation of the `distinctive features` function as part of the previous AST:

```
AST Node: SelectByPredicateAndOrder
    Parameters:
        - TableDescription (string) = "information about pizzerias"
        - PredicateDescription (string) = "filter pizzerias by profitability"
        - SortOrderDescription (string) = "order by pizzeria count"
    Subtree:
        - AST Node: Filter
            Parameters:
                - FieldName (string) = "profitability"
                - Predicate (string) = "greater than zero"
            Subtree:
                - AST Node: Table
                    Parameters:
                        - TableName (string) = "pizzerias"
                    Subtree:
                        - AST Node: Field
                            Parameters:
                                - FieldName (string) = "profitability"
        - AST Node: Aggregate
            Parameters:
                - AggregateFunction (string) = "distinct"
                - ParameterName (string) = "distinctive features"
            Subtree:
                - AST Node: Compute
                    Parameters:
                        - CalculationFunction (string) = "distinctive features"
                    Subtree:
                        - AST Node: Table
                            Parameters:
                                - TableName (string) = "pizzerias"
                            Subtree:
                                - AST Node: Field
                                    Parameters:
                                        - FieldName (string) = "features"
        - AST Node: Sort
            Parameters:
                - FieldName (string) = "pizzeria count"
                - SortOrder (string) = "descending"
            Subtree:
                - AST Node: Aggregate
                    Parameters:
                        - AggregateFunction (string) = "count"
                        - ParameterName (string) = "pizzeria count"
                    Subtree:
                        - AST Node: Table
                            Parameters:
                                - TableName (string) = "pizzerias"
                            Subtree:
                                - AST Node: Parameter
                                    Parameters:
                                        - ParameterName (string) = "distinctive features"
        - AST Node: Function
            Parameters:
                - FunctionName (
````

**Expected behavior**
All of the output needs to be shown in case incomplete markdown returned, no matter that markdown is malformed. If it is malformed, it should be rendered without formatting at all, that what I would expect.

In this case the problem is caused by the fact that chatgpt could not complete the output, probably a length restriction, and the final three backtick sequence ` ``` ` was not written. Nevertheless it should not break the app. At least I would like to see all of the output even if it is broken markdown. After all, I could ask chatgpt to generate some text with backticks.

**Screenshots**
<img width="986" alt="image" src="https://user-images.githubusercontent.com/2792885/234151674-685c54ab-0486-4995-a80b-fb86f45ff549.png">

**Environment:**
 - OS version: [e.g. macOS Ventura 13.0.1]
arm64

**Comment by Renset** [comment]
@george-polevoy thanks for providing such a detailed bug report! The cause of this issue is clear to me and I'm planning to include the fix in the next update

**Comment by Renset** [comment]
@george-polevoy may I include your code output from the above in unit tests and commit it?

**Comment by Renset** [comment]
@george-polevoy fixed in [v1.1.0](https://github.com/Renset/macai/releases/tag/v1.1.0)
